scripts/base/frameworks/cluster/main.zeek

```diff
@@ -241,6 +257,36 @@ export {
 	## locally whenever a connected cluster node becomes disconnected.
 	global node_down: event(name: string, id: string);
 
+	## When using broker-enabled cluster framework, this event will be sent to
+	## the manager once a cluster node has set up all its outgoing connections
+	## to other cluster nodes based on the given cluster layout.
+	##
+	## name: The name of the now fully connected node.
+	##
+	## id: The identifier of the now fully connected node.
+	##
+	## redispatched: If true, the node has previously signaled that it is fully
+	##               connected. This may happen in case the manager restarts.
+	##
+	## .. warning::
+	##
+	##     There is no tracking of cluster node connectivity. Thus, there is
+	##     no guarantee that all peerings still exist at the time of this event
+	##     being raised.
+	global node_fully_connected: event(name: string, id: string, redispatched: bool);
```

Comment:
> 	redispatched: If true, the node has previously signaled that it is fully
>	               connected. This may happen in case the manager restarts.

The implementation looks like it's currently passing `seen_cluster_connected` and that seems needed, but the docstring disagrees. I like "seen_cluster_connected" over "dispatched", too, as the current `redispatched` is closer to "sending a second time" which isn't the same (and indeed isn't good enough for what it's being used for).


I'm also forth and back if we should expose `node_fully_connected` as is. It seems a nice primitive to have, but  the `redispatched / seen_cluster_connected` might be confusing to the "local user". I think it's also raised multiple times locally for a restarting manager.

This could be fixed by separating the local event and the one we send to the manager and letting go of auto_publish.

Essentially, the same motivation as for `cluster_connected`: Ensure it is raised once as a local event and no extra info that would allow advanced logic. But I might be too biased here.

Reply:
So this is essentially a vote to rename `redispatched` to `seen_cluster_connected`, correct?

Regarding splitting the events: I am not a fan of wrapping users in cotton. Exposing it all provides greater flexibility. If needed, one could just check `name` or `seen_cluster_connected`. Writing Zeek scripts I never found myself grumbling that some information I am not interested in was exposed. Unfortunately, the inverse has happened here and there.

Reply:
Turned out that `redispatch` just didn't click so we settled on `resent`.